test/integration/context-based-restrictions.v1.test.js

```diff
@@ -580,6 +581,77 @@ describe('ContextBasedRestrictionsV1_integration', () => {
     });
   });
 
+  test('listRules() - List rule with valid service_group_id attribute', async () => {
+    //create new rule with service_group_id attribute
+    const ruleContextAttributeModel = {
+      name: 'networkZoneId',
+      value: 'af03b24cbaed353f53d2290a9123a54a',
+    };
+
+    const ruleContextModel = {
+      attributes: [ruleContextAttributeModel],
+    };
+
+    const resourceAttributeAccountIdModel = {
+      name: 'accountId',
+      value: accountId,
+      operator: 'stringEquals',
+    };
+
+    const resourceAttributeServiceGroupIDModel = {
+      name: 'service_group_id',
+      value: 'IAM',
+    };
+
+    const resourceModel = {
+      attributes: [resourceAttributeAccountIdModel, resourceAttributeServiceGroupIDModel],
+    };
+
+    const params = {
+      contexts: [ruleContextModel],
+      resources: [resourceModel],
+      description: 'this is an example of rule with a service_group_id',
+      enforcementMode: ContextBasedRestrictionsV1.CreateRuleConstants.EnforcementMode.ENABLED,
+      transactionId: uuidv4(),
+    };
+
+    let res;
+    try {
+      res = await contextBasedRestrictionsService.createRule(params);
+    } catch (err) {
+      console.warn(err);
+    }
+
+    expect(res).toBeDefined();
+    expect(res.status).toBe(201);
+    expect(res.result).toBeDefined();
+
+    ruleId = res.result.id;
+    ruleEtag = res.headers.etag;
+
+    //list rule with service_group_id attribute
+    const listParams = {
```

Comment:
I would expect that we'd want to pass the serviceGroupId parameter to make sure filtering by the property is working as expected?